CODE_OF_CONDUCT.adoc

```diff
@@ -0,0 +1,52 @@
+# Code of Conduct
+
+The Eclipse-uProtocol project is intended to be used in production environments (for automotive and other use cases). In order to ensure the project is well maintained and supported, we have outlined additional expectations for committers and project leads than what is currently defined in the following the Eclipse Resources:
+
+- https://raw.githubusercontent.com/eclipse/.github/master/CODE_OF_CONDUCT.md[Eclipse Code of Conduct]
+- https://www.eclipse.org/projects/handbook/[Eclipse Foundation Handbook]
+- https://gitlab.eclipse.org/eclipsefdn/emo-team/policies/legal-faqs/-/blob/main/documents/src/adoc/committers-dd.adoc[ Eclipse Committer Due Diligence Guidelines]
+
+Although many of the items discussed in this document is intuitive, we ask that you read and understand them anyways.
+
+
+We (the committers) aim to run this project in an open and fair manner where contributions are encouraged from all companies and individuals. If you happen to notice any violations to this code of conduct, please contact the project lead and/or the Eclipse Foundation Management Office (EMO).
+
+
+## Committers
+https://www.eclipse.org/projects/handbook/#roles-cm[Eclipse-uProtocol Committers] play a vital role to ensure ensure contributions from others (and themselves) follow the vision and mission of the project, and adhere to this code of conduct. 
+In the next section we will elaborate on their expected area of expertise and responsibilities, along with how committers are nominated and retired.
+
+[#area-of-expertise]
+### Committer's Area of Expertise
+Committers are chosen based on merit and the area of expertise that they can steward (maintain). List below is not exhaustive and shall update as the project evolves:
+
+1. **Specifications** (up-spec, up-core-api): Ensuring the vision/mission of the project is upheld, process is followed, and the specifications are maintained and updated. Concerns are discussed and addressed w.r.t backwards compatibility support and new features.
+2. **Programming language:** Experts in a particular language who help to review and comment on contributions for a given supported language (Java, Kotlin, Rust, C++, Python, etc...)
+3. **Implementation:** Experts in a particular implementation of uProtocol (ex. Android, Zenoh, Sommr/SOME/ip, Cloud, etc...)
```

Comment:
previous item (programming languages) included reference to expected contributions, while this one only lists the expected profile of committers. 
Description should be consistent for all 3 types: either only profile, or profile and expected contributions
Given the additions that follow, maybe redefine this section to focus on profile only (and explicitly state that this part defines the profile?)